### PR TITLE
Add logic for deleting comments

### PR DESCRIPTION
**Pull request by LudvigHz**

### CODE REVIEW

File: lego/apps/comments/models.py [hunk 1]
```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
+            self.created_by = None
+            self.text = "<p>Slettet</p>"
```
Comment (odinuge) [inline lego/apps/comments/models.py]: Does it make more sense to set this to `null`, and handle it in the frontend? Kinda hard to change the behavior if we save this string in the db.

(I have no idea if our `ContentField` will handle it properly without modification tho.)

File: lego/apps/comments/models.py [hunk 2]
```diff
@@ -27,5 +27,14 @@ class Meta:
...
+            if Comment.objects.filter(parent=self).exists():
+                self.save()
+            else:
+                super(Comment, self).delete()
```
Comment (odinuge) [inline lego/apps/comments/models.py]: Does it make more sense to do `delete(force=True)`. I don't see why this should be a `PersistentModel` at all

Comment (odinuge) [reply lego/apps/comments/models.py]: Also, why set `created_by` and `text` before deleting (when using force)? Isn't it more logical to do that separatly? And, if soft deleting, does `.delete()` save the new values?

File: lego/apps/comments/models.py [hunk 3]
```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
+            self.created_by = None
+            self.text = "<p>Slettet</p>"
+            if Comment.objects.filter(parent=self).exists():
```
Comment (odinuge) [inline lego/apps/comments/models.py]: Nice :100:

File: lego/apps/comments/models.py [hunk 4]
```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
```
Comment (orhanhenrik) [inline lego/apps/comments/models.py]: Is there any reason to open a transaction here?

Comment (LudvigHz) [reply lego/apps/comments/models.py]: I guess not really.

File: lego/apps/comments/models.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 from django.contrib.contenttypes.fields import GenericForeignKey
 from django.contrib.contenttypes.models import ContentType
-from django.db import models
+from django.db import models, transaction
```
Comment (odinuge) [inline lego/apps/comments/models.py]: Is this import still used?

File: lego/apps/comments/models.py [hunk 6]
```diff
@@ -15,7 +15,7 @@ def get_queryset(self):
 
 class Comment(BasisModel):
 
-    text = ContentField(allow_images=False)
+    text = ContentField(allow_images=False, null=True)
```
Comment (odinuge) [inline lego/apps/comments/models.py]: Have you tested this (both frontend and backend)?

Comment (LudvigHz) [reply lego/apps/comments/models.py]: Tested frontend, but can write a test for backend as well.

**Pull request merged**